src/test/java/teammates/test/cases/ui/InstructorFeedbackSubmissionEditSaveActionTest.java

```diff
@@ -29,73 +29,74 @@
 import teammates.ui.controller.RedirectResult;
 
 public class InstructorFeedbackSubmissionEditSaveActionTest extends BaseActionTest {
-
     private static final DataBundle dataBundle = loadDataBundle("/InstructorFeedbackSubmissionEditSaveActionTest.json");
-    
+
     @BeforeClass
     public static void classSetUp() throws Exception {
         printTestClassHeader();
         removeAndRestoreDatastoreFromJson("/InstructorFeedbackSubmissionEditSaveActionTest.json");
         uri = Const.ActionURIs.INSTRUCTOR_FEEDBACK_SUBMISSION_EDIT_SAVE;
     }
-    
+
     @AfterClass
     public static void classTearDown() throws Exception {
         new BackDoorLogic().removeDataBundle(dataBundle);
     }
-    
+
     @Test
     public void testExecuteAndPostProcess() throws Exception {
         InstructorAttributes instructor1InCourse1 = dataBundle.instructors.get("instructor1InCourse1");
         gaeSimulation.loginAsInstructor(instructor1InCourse1.googleId);
-        
+
         ______TS("Unsuccessful case: test empty feedback session name parameter");
-        
+
         String[] submissionParams = new String[]{
                 Const.ParamsNames.COURSE_ID, dataBundle.feedbackResponses.get("response1ForQ1S1C1").courseId
         };
-        
+
         InstructorFeedbackSubmissionEditSaveAction a;
         RedirectResult r;
-        
+
         try {
             a = getAction(submissionParams);
             r = (RedirectResult) a.executeAndPostProcess();
             signalFailureToDetectException("Did not detect that parameters are null.");
         } catch (NullPostParameterException e) {
-            assertEquals(String.format(Const.StatusCodes.NULL_POST_PARAMETER, 
-                    Const.ParamsNames.FEEDBACK_SESSION_NAME), e.getMessage());
+            assertEquals(String.format(Const.StatusCodes.NULL_POST_PARAMETER,
+                                       Const.ParamsNames.FEEDBACK_SESSION_NAME), e.getMessage());
```

Comment:
Didn't change it eventually cause the first one has 2 lines (3 on 2nd, 2 on 3rd) and can be seen without scrolling (3rd one > 110 by abit due to try catch block and indents due to method declaration) but I see why they might be better. Eventually we might want to follow either the first or the 2nd one, some parts of the code base we have String.format(4 different constants) and it does quite get long so 3rd option might not be consistent enough.